### DISCUSSION TITLE
关于业务模块划分

### DISCUSSION THREAD

**Discussion by getda** [mineadmin/MineAdmin#286] [General]
想看看大家对于小程序、APP相关接口是怎么做的呢

**Comment by zds-s** [comment]
1. 单独起一个项目，中台是中台。api 是 api.  一般单体架构不用这样，维护起来比较麻烦
2. 在 app/ 下新建模块，比如小程序就是 `/project/app/miniApp` 模块 app 就是 `/project/app/Business` 模块。以此类推，或者对外接口全部放到一个 `/project/app/Business` 模块中
3. 使用 `/project/api/` 建设对外接口，但我个人不建议用这个

**Reply by getda** [reply]
好的 感谢分享
